content/system76-software.md

```diff
@@ -199,7 +199,7 @@ Now log out and use the <u>Extensions</u> to enable the extenstion.
 If `Gio.DBusError` presents itself when attempting to enable the extension within the Extensions application, confirm that the `system76-power.service` is running:
```

Comment:
The old/alias name is still being used on this line.

Reply:
Nice catch. I believe https://github.com/system76/docs/pull/1073/commits/8f7f9f1143403ead1cfb6e11a840698a04976086 resolves the last two instances of the old name.